### DISCUSSION TITLE
Github linguist color change to orange

### DISCUSSION THREAD

**Discussion by Teddy55Codes** [ocaml/ocaml#12013] [Polls]
# GitHub color change from `#3be133` to `#ee6a1a`

## What is this all about
The [github/linguist](https://github.com/github/linguist) repository is used to provide syntax highlighting and identification for many languages on github including OCaml. 

## Why?
The color for OCaml was set in 2012 with the [Colors! commit](https://github.com/github/linguist/commit/c3d6fc5a) (Back when the color feature was originally introduced.). The color choosen back then was `#3be133` (as you probably know :) ). I think that is not very fitting. The OCaml logo, camels and desert sand are all orange. This is why i propose this color change. 

The ongoing pull request with more information can be found [here](https://github.com/github/linguist/pull/6284)

**Comment by gasche** [comment]
I forwarded this poll [on Discuss](https://discuss.ocaml.org/t/the-color-associated-with-ocaml-on-github-is-bad-lets-make-it-good/5597/19?u=gasche) so that more people see it.

**Comment by Teddy55Codes** [comment]
Ok, i think the result of the poll is very clear. If the result stays like this i will create a new poll in a day or two on the exact shade of orange that will be choosen.
The current candidates are:
* `#ef7a08`
* `#ee6a1a`

If you would prefer a different one please reply with your shade and i will include it in the poll.

###### Edit
To be clear creating the other poll doesn't mean that i assume this one has concluded.

**Comment by Octachron** [comment]
A result cannot be very clear in 4 hours. Taking in account time zones and general availability of people, you should wait at least one week before deciding that you have sampled ... some subset of people lurking on GitHub?

Without a more explicit polling strategy, I fear that you cannot conclude more than there might be some interest in a change of the `linguist` color for OCaml.

**Reply by Teddy55Codes** [reply]
Ok, yes that i probably better i will wait until next week and if the result is still in favour of changing the color i will create the next poll.

**Reply by Teddy55Codes** [reply]
I'm sorry that my phrasing was not that exact but i did not mean to imply that this poll would be done if the other one is created. I don't have the permissions to do that anyway.

**Comment by Teddy55Codes** [comment]
The color change has been approved ([pull request](https://github.com/github-linguist/linguist/pull/6284)) and will be part of the next linguist release.

**Reply by Octachron** [reply]
I am sorry, but why did you go ahead with the proposal with a lone poll as "evidence" of explicit support ?

**Reply by Teddy55Codes** [reply]
I went ahead because the response was very positive. I also made it clear from the start that my intention was to change the color if the community feed back was positive. I have linked the ongoing pull request from the start. Also this poll has started in February i taught that if the color change was a big problem i would have gotten a lot more push back.

**Reply by Octachron** [reply]
Personally, I consider that twenty positive reactions over a half-thousand regularly active OCaml users on discuss.ocaml.org is more a mostly indifferent reaction with a small core of positive reactions.

But since there were no push-back at all (this time), let's go with the change for now.

**Reply by avsm** [reply]
It's generally very difficult to get any strong positive reactions out of the OCaml community.  This is probably the best we'll get back short of doing polls at in-person meetups, which will take longer.  I'm personally largely indifferent to the change (but appreciate someone else thinking about it, like @Teddy55Codes).
